src/Transports/MassTransit.ActiveMqTransport/ActiveMqTransport/Middleware/ActiveMqConsumer.cs

```diff
@@ -68,6 +69,9 @@ void HandleMessage(IMessage message)
                     context.Dispose();
                 }
             }, Stopping);
+
+            if (!result.IsCompletedSuccessfully())
```

Comment:
@phatboyg what do you think about this? seems like the single usage of sync flow

Reply:
this will have similar behavior, and for happy cases is going to use `TryWrite` which should return a result (when channel is not full)

Reply:
decided to keep `PushWithWait`